packages/ses/src/module-static-record.js

```diff
@@ -0,0 +1,39 @@
+import * as babel from '@agoric/babel-standalone';
+// We are using the above import form, and referring to its default export
+// explicitly below like babel.default, because the proper construct causes a
+// Rollup error.
+// This form:
+//   import babel from '@agoric/babel-standalone';
+// And this variant:
+//   import { default as babel } from '@agoric/babel-standalone';
+// Both produce:
+//   Error: 'default' is not exported by .../@agoric/babel-standalone/babel.js
+import { makeModuleAnalyzer } from '@agoric/transform-module';
+
+const analyze = makeModuleAnalyzer(babel.default);
+
+// moduleAnalyses are the private data of a ModuleStaticRecord.
+// We use moduleAnalyses in the loader/linker to look up
+// the analysis corresponding to any ModuleStaticRecord constructed by an
+// importHook.
+// This WeakMap is shared among modules in this package, but not exposed to the
```

Comment:
The ModuleStaticRecord constructor sets, the linker gets. No other parties have access to the WeakMap. The package.json limits visibility of module exports to just those exported by src/main.js. I will note this.

Reply:
In that case, I'd prefer we only export the ability to get.

Re `package.json`. what about on the browser or XS?

Reply:
I’m electing to merge module-static-record.js into compartment-shim.js to avoid the need to communicate this between modules. I don’t think it’s a problem, but this will eliminate cause for concern.

Reply:
I agree with all this. Although in our overall system we likely could ensure and explain why this is not a concern, I'd much rather just eliminate the concern. Thanks.

Reply:
Looked. Resolving.